iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -66,15 +66,9 @@ class LLVMAOTTargetBackend final : public TargetBackend {
       : options_(std::move(options)) {}
 
   // NOTE: we could vary these based on the options, such as by arch/etc.
-  std::string name() const override { return "llvm_aot"; }
-  std::string filter_pattern() const override {
-    llvm::Triple targetTriple(options_.targetTriple);
-    if (targetTriple.isWasm()) {
-      return "wasm*";
```

Comment:
putting "wasm" in the filter/deviceID was load bearing in my work to get wasm running, but I guess we can drop that until the wasm work is fully landed

Reply:
this is all changing, and this particular piece is the simplest kind of work that can be done on bringup so not worth preserving when unused (YAGNI)